### PR TITLE
Add basic UI 

### PR DESCRIPTION
**Pull request by codecaiine**
- Created a main.rb file that serves as console app entry-point.
- Created menu list in main.rb

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    print 'Age: '
+    age = gets.chomp.to_i
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp.downcase
+
+    student = Student.new(age, @class, name, parent_permission)
```
Comment (o-rashad) [inline main.rb:99]: Well done creating all these interacting methods :100: ,
Unfortunately i ran into this error when i try to create a `student` instance :disappointed:  
> FireShot Capture 004 - main rb - oop-school-library - Gitpod Code_ - copper-sparrow-66g6q7js ws-eu18 gitpod io

- - [x] Investigate the error and solve the issue. :+1:

**Review by o-rashad — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️

Asslam Alikom @codecaiine  👋😄,
I'm Omar and I will be your reviewer for this project 😊

Good job so far!👌😃, There are some issues 🐞 that you still need to work on to go to the next project but you are almost there! 🏁🏁🏁

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (codecaiine) [reply main.rb:99]: Alaikum salam @od-c0d3r
Nice to meet you Omar and thank you for reviewing my code and for your comments. I was able to implement all required changes as you can see below:
teseeeet
T2
T1
TESSSSS

**Review by mke2111 — APPROVED** [review]
### Hi @codecaiine, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by codecaiine** [comment]
Hi @mke2111 ,

Thank you for reviewing my work and approving it.

Cheers!

File: main.rb (hunk 1 above)
Comment (od-c0d3r) [reply main.rb:99]: Thanks @codecaiine :wave: for your feedback.
Sorry for the inconvenience.

- [ ] Now I need to investigate my error and solve it :sweat_smile:

Comment (o-rashad) [reply main.rb:99]: @codecaiine Keep it up :100: :chart:

Comment (codecaiine) [reply main.rb:99]: Thanks Omar :)

**Pull request merged**